README.md

```
@@ -1 +1,60 @@
+make prod
+```
+
+### Run the Playground
+
+```bash
+cd playground
+bun install
+bun run dev
+```
+
+Open http://localhost:5173 in your browser.
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)